docs/README-migration.md

```diff
@@ -446,3 +446,15 @@ Programs which have access to shaders can implement more robust versions of thos
 Removed 'SDL_GL_CONTEXT_EGL' from OpenGL configuration attributes
 You can instead use 'SDL_GL_SetAttribute(SDL_GL_CONTEXT_PROFILE_MASK, SDL_GL_CONTEXT_PROFILE_ES);'
 
+SDL_VideoInit() and SDL_VideoQuit() have been removed. Instead you can call SDL_InitSubSytem() and SDL_QuitSubSytem() with SDL_VIDEO_AUDIO, which will properly refcount the subsystems.
```

Comment:
Lots of whitespace here, and a typo? I did a little cleanup in the audio section, maybe you can copy that?

Reply:
yes, done!